Lib/test/test_subprocess.py

```diff
@@ -661,6 +666,26 @@ def test_stdin_devnull(self):
         p.wait()
         self.assertEqual(p.stdin, None)
 
+    def test_pipesizes(self):
```

Comment:
Also add a test_pipesize_default that confirms that passing in -1 does not call fcntl.

Reply:
:+1: